src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -18,58 +29,66 @@
   }
 }
 
-.nav__search input {
-  width: 18.125rem;
-  text-align: left;
-  background-color: $saturated-black;
-  border: solid 2px rgba(0, 0, 0, 0.13);
-  font-size: 0.9em;
-}
-
-.nav-search-results {
-  z-index: z(navbar-nav-search-results);
-
+.nav-dropdown {
   position: fixed;
-  top: 60px;
-
-  min-width: 18.125rem;
-
+  z-index: 1;
```

Comment:
Do we need to set a `z-index` here? (often we don't actually need them and just end up overusing them) If so, we should be using our `z()` helper (defined in `constants`)

Reply:
yeah, you are right. We don't need z-index here, although we need it for search-input (see my explanation copied from another comment below)
fixed: https://github.com/sparkletown/sparkle/pull/1328/commits/e21e67445665543be0fab0f660743356141c30c7

> I used z-index to have input-div above the search-results. Search results are going from the top of the screen and if they both are on the same z-index level search-results overlaps input-div for a short period when appearing and hiding.